src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,377 @@
+    name: Ian Simpson
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: isimpson@ripple.com
+    uri: https://www.ripple.com
+
+normative:
+    RFC2119:
+    RFC2818:
+    RFC8446:
+    RFC7231:
+    RFC7413:
+    RFC6265:
+    RFC6570:
+    PAYID-URI:
+      title: "The 'payid' URI Scheme"
+      target: https://tbd.example.com/
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+    PAYID-DISCOVERY:
+      title: "PayID Discovery"
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+
+informative:
+    RFC5988:
+
+--- note_Feedback
+
+This specification is a draft proposal, and is part of the [PayID Protocol](https://payid.org/) initiative. Feedback related to this document should be sent in the form of a Github issue at: https://github.com/payid-org/rfcs/issues.
+ 
+--- abstract
+This specification formalizes how a payment recipient, such as a merchant or a non-profit, can automatically 
+initiate a payment from a customer using only the customer's PayID. 
+
+--- middle
+
+# Terminology
+
+This protocol can be referred to as the `PayId Easy Checkout Protocol`. It uses the following terminology:
+   
+* PayID client: a client that queries a PayID server using the PayID Protocol as defined in [PAYID-PROTOCOL][].
+* PayID server: the endpoint that returns payment account(s) information, which conforms to the PayID Protocol.
+* merchant: individual or entity receiving the payment (ie e-commerce merchant, charity).
+* customer: individual or entity originating the payment to the `merchant`.
+* wallet: the holder of funds for the `sender` (may or may not be custodied).
+* Redirect URL: The URL that is the result of the PayID Easy Checkout protocol, and which can be used to redirect a client to a wallet corresponding to a particular PayID.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol designed to provide a standard method for online merchants to
+request a payment from a customer using only the customer's PayID. Implementations
+of the protocol should require little to no server-side engineering efforts, while creating an improved and uniform
+user experience for customers.
+
+The main focus of the Protocol is on PayID Easy Checkout Discovery, which defines how a PayID client can use a PayID
+to retrieve a PayID Easy Checkout URI Template which, when expanded, constitutes a PayID Easy Checkout URL 
+representing a resource that the customer's digital wallet can use to initiate a payment to the merchant. 
+
+Though section (TODO: link to appendix example usage section) of this specification provides an example usage of a 
```

Comment:
You can link to the other sections in the RFC. Checkout this example [here](https://github.com/rfcs/crypto-conditions/blame/master/src/spec/crypto-conditions.md#L295).

Basically, you take the heading title (e.g., `PayID Easy Checkout Protocol`) and then transpose it to a hyphenated+lower-case style (e.g., `payid-easy-checkout-protocol`) and then put them together in a link, like this:

```
See [this section here](#payid-easy-checkout-protocol).
```

One thing to be careful of is if you change the title of a heading but forget to updates other links in the document.

Reply:
Sweet. I was going to wait until we finalized everything to manually add sections, but this is better